sdk/examples/xo_python/sawtooth_xo/processor/handler.py

```diff
@@ -21,16 +21,47 @@
 from sawtooth_sdk.processor.exceptions import InternalError
 from sawtooth_sdk.protobuf.transaction_pb2 import TransactionHeader
 
+
 LOGGER = logging.getLogger(__name__)
 
 
-class XoTransactionHandler:
-    def __init__(self, namespace_prefix):
-        self._namespace_prefix = namespace_prefix
+# namespace
+FAMILY_NAME = 'xo'
+XO_NAMESPACE = hashlib.sha512(FAMILY_NAME.encode('utf-8')).hexdigest()[:6]
+
+def make_xo_address(name):
+    return XO_NAMESPACE + hashlib.sha512(name.encode('utf-8')).hexdigest()[-64:]
+
+# encodings
+def decode_txn_payload(payload):
+    return payload.decode().split(',')
+
+def decode_state_data(data):
+    return data.decode().split(',')
+
+def encode_state_data(data):
+    return ','.join(data).encode()
 
+# board
+SIDE_LENGTH = 3
+SPACE_COUNT = SIDE_LENGTH ** 2
+VALID_SPACES = range(1, SPACE_COUNT + 1)
+VALID_MARKS = MARK_1, MARK_2, EMPTY_SPACE = 'X', 'O', '-'
```

Comment:
Should they be renamed PLAYER_1_MARK, etc? They're the marks for players 1 and 2. They happen to be 'X' and 'O', but they could be different.